App.js

```diff
@@ -252,31 +656,52 @@ export default function App() {
         </View>
         <View
           style={
-            endOfGame
+            endOfGame || numberOfPlayers === 0
               ? styles.itemInvisible
               : odd
-              ? [styles.item, styles.markSpot1]
+              ? [styles.item, styles.markSpotRed]
               : [styles.item, styles.itemBlue]
           }
         >
           <TouchableHighlight onPress={() => pressOdd()}>
             <Text style={styles.itemText}>Odd</Text>
           </TouchableHighlight>
         </View>
-        <Button onPress={() => pressRoll()} title="Roll" color="blue" disabled={endOfGame ? true : false} />
-        <View style={endOfGame ? styles.itemInvisible : styles.itemNav}>
+        <View style={endOfGame || numberOfPlayers === 0 ? styles.itemInvisible : null}>
+          <Button
+            onPress={() => pressRoll()}
```

Comment:
You don't need the `() =>` here, you can just pass pressRoll directly

Reply:
Done.